.github/workflows/iceoryx.yml

```diff
@@ -1,33 +0,0 @@
-name: Build iceoryx
```

Comment:
Why have you deleted it?  because it's strange to have a workflow for iceoryx here. @Karsten1987 what was the reason for having it? Just to check if iceoryx master currently works. Which is maybe obsolete now as we use the stable release

Reply:
I just followed what @Karsten1987 did in #37 and #43, and I will keep the change in this foxy branch.